platform/android/app/build.gradle

```diff
@@ -110,43 +116,43 @@ dependencies {
     implementation project(":core")
     implementation 'org.msgpack:msgpack-core:0.8.20'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
-    implementation 'androidx.appcompat:appcompat:1.2.0'
-    implementation 'androidx.core:core-ktx:1.3.1'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.1'
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0'
-    implementation 'androidx.navigation:navigation-fragment-ktx:2.3.0'
-    implementation 'androidx.navigation:navigation-ui-ktx:2.3.0'
-    implementation 'androidx.recyclerview:recyclerview:1.1.0'
-    implementation "androidx.webkit:webkit:1.3.0"
-    implementation "androidx.preference:preference-ktx:1.1.1"
-    implementation 'com.google.android.material:material:1.2.0'
+    implementation 'androidx.appcompat:appcompat:1.4.2'
+    implementation 'androidx.core:core-ktx:1.8.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.4.1'
+    implementation 'androidx.navigation:navigation-fragment-ktx:2.4.2'
+    implementation 'androidx.navigation:navigation-ui-ktx:2.4.2'
+    implementation 'androidx.recyclerview:recyclerview:1.2.1'
+    implementation "androidx.webkit:webkit:1.4.0"
+    implementation "androidx.preference:preference-ktx:1.2.0"
+    implementation 'com.google.android.material:material:1.6.1'
     implementation 'org.slf4j:slf4j-api:1.7.30'
     implementation 'com.synnapps:carouselview:0.1.5'
     implementation 'com.google.code.gson:gson:2.8.9'
     implementation group: 'org.apache.jena', name: 'jena-core', version: '3.17.0'
```

Comment:
Just in case you're interested, we need to peg this to that version, although there're some (mostly irrelevant) security issues. Check out #420

Reply:
🤔 I don't get it. Do we need to fix some dependencies to a certain version due to security reasons?

Reply:
No, we don't. Just explains why jena is at such an outdated version.